mariner/server/__init__.py

```diff
@@ -25,9 +25,9 @@ def index() -> str:
 class CacheBootstrapper(multiprocessing.Process):
     def run(self) -> None:
         os.nice(5)
-        for file in FILES_DIRECTORY.rglob("*.ctb"):
+        for file in FILES_DIRECTORY.rglob("*.ctb" or "*.cbddlp"):
```

Comment:
This won't work... `or`ing two strings that are true-ish will cause this to just pass `"*.ctb"` to `rglob`.

The way to go is to use `itertools.chain` since `rglob` doesn't have any way of matching the two extensions at once:

```python
from itertools import chain

# ...

for file in chain(FILES_DIRECTORY.rglob("*.ctb"), FILES_DIRECTORY.rglob("*.cbddlp")):
    # ...
```

Reply:
code updated